build/bin/release

```diff
@@ -0,0 +1,47 @@
+#!/bin/bash
+
+if [ "$#" -lt "1" ]; then
+    echo "must specify chart directory"
+    exit 1
+fi
+
+set -x
+
+chart=$1
+
+rm -r build/release
+
+owner=${GITHUB_USER:-onosproject}
+repo=${GITHUB_REPO:-onos-helm-charts}
+remote=${GIT_REMOTE:-origin}
+branch=$(git branch --show-current)
+
+# Package the Helm chart
+helm package $chart --destination build/release
+
+# Upload the Helm chart release
+cr upload \
+    --git-base-url https://api.github.com/ \
+    --git-upload-url https://uploads.github.com/ \
+    --owner $owner \
+    --git-repo $repo \
+    --package-path build/release
+
+# Switch to the gh-pages branch
+git checkout gh-pages
+
+# Update the repository index
+cr index \
+    --index-path index.yaml \
+    --git-base-url https://api.github.com/ \
+    --git-upload-url https://uploads.github.com/ \
+    --owner $owner \
+    --git-repo $repo \
+    --charts-repo https://$owner.github.io/$repo \
+    --package-path build/release
+
+# Commit the updated index.yaml
+git commit -a -m "Add $chart release to index.yaml"
+git push $remote gh-pages
```

Comment:
Should we do git push --force here to remove the history like what Travis deploy command does? I don't know if it is important to keep history of commits for pushing index.yaml file.

Reply:
I think it's important considering it's how @zdw recognized a caching issue that could result from the changes.